department-of-veterans-affairs/va.gov-cms
# VACMS-17545: Update Service Taxonomy view

## Description

Relates to #17545

## Testing done
Manual in tugboat

## Screenshots

![Screenshot 2024-09-12 at 5 39 13 PM](https://github.com/user-attachments/assets/e66071c9-a53b-4938-a16f-ffac02dcc889)


## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As any authenticated user 
1. Log in to [the tugboat](https://pr19191-lrmpkpcahzfciprtpevvgorq6d8rgxjn.ci.cms.va.gov/)
2. Go to Content > [VA Services](https://pr19191-lrmpkpcahzfciprtpevvgorq6d8rgxjn.ci.cms.va.gov/va-services)
   - [x] Confirm that the Stop Code column no longer appears
   - [x] Confirm that the view can be sorted by different columns
3. Select 'VBA facility service' from the Content Type drop down menu
   - [x] Confirm that all services that show up below all have the VBA descriptions and VBA types of care
4. Select 'VAMC system health service' from the Content Type drop down menu
   - [x] Confirm that all services that show up below all have VAMC descriptions and types of care
   - [x] Confirm that some services have VAMC patient friendly names and VAMC common conditions
5. Select 'Vet Center facility service' from the Content Type drop down menu
   - [x] Confirm that all services that show up below all have Vet Center descriptions and types of care
   - [x] Confirm that some services have Vet Center patient friendly names and Vet Center common conditions
   - [x] Confirm that some services are marked as Required Vet Center Services
6. Compare against the current view on Prod https://prod.cms.va.gov/va-services
   - [x] Notice that VAMC descriptions no longer have a line break between the label and the description
   - [x] Notice that the VAMC and Vet Center labels have been shorted in the columns: Patient friendly name, common conditions, description, and type of care 

### Definition of Done
